### PR TITLE
Create doc for Governance Review Week

### PR DESCRIPTION
**Pull request by OrkoHunter**
**Write path(s) which will be affected by this Pull Request.**

- `/community/governance-review-week`

**Justify your changes or addition.**
Described Governance Review Week
Wrote about its structure and timeline
Wrote about its purpose
Added links and questions

Governance Review Week generated a lot of value in the last two years. The questions were discussed were thought carefully and answered sincerely.

**Why do you think it should be documented?**
1. New members will already know about the event and its history.
2. It will be easier to organize in the future by looking at the archives.
3. Other societies can learn from our system. (TLS already does).

Co-Authored-By: Ayush Kaushal <ayushk4@gmail.com>

### CODE REVIEW

File: docs/community/governance-review-week.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
+# Governance Review Week
+
+# What is it?
+
+It is a week dedicated to the Governance of KOSS.
+
+Our intention behind organizing the Governance Review Week is to help the core team members measure their level of involvement in KOSS. The week also promotes open discussions between Seniors and the Core team members. We want each one of our members to be frank with their batchmates as well as seniors in sharing their opinions and perspective on every aspect of KOSS.
+
+We also review how KOSS performed as a team in the past year. We identify our problems and discuss solutions. We want to make sure we eliminite all forms of toxicity from KOSS (It can be habits of how we do things, or some Core Team members themselves.)
```
Comment (Ayushk4) [inline docs/community/governance-review-week.md]: A Grammarly check might be good on the entire document: `eliminite` -> `eliminate`

**Comment by OrkoHunter** [comment]
Thank you so much @Ayushk4 for the comment! My code editor's extension did not catch it. I will get it reviewed by Grammarly from the next time. Ironically, I wrote it down today - https://github.com/kossiitkgp/docs/blob/master/.github/CONTRIBUTING.md#conventions 😅 

Pushed a new commit. The person merging this PR should "Squash and Merge"

**Comment by sangeet259** [comment]
I see why you edited the wiki ;) @OrkoHunter

**Comment by sangeet259** [comment]
All fine, apart from the fact that the videos are on the account which we are in the process of ditching.

**Comment by OrkoHunter** [comment]
Note: At the end of the review week, a new version of the doc is written. The changelog is described and published. (Alumni are interested in this).

**Comment by OrkoHunter** [comment]
This is ready to merge. (Commits should be squashed into 1)

**Pull request merged**